translations/ja/LC_MESSAGES/messages.po

```diff
@@ -654,7 +654,7 @@ msgstr ""
 #: /content/includes/menu.yaml /content/pages/home.html:37
 #: /content/pages/home.html:77 /views/amp-ads.html:409 /views/amp-ads.html:444
 msgid "News"
-msgstr ""
+msgstr "ブログ"
```

Comment:
This translation are not bad translation. because `news` menu will move in `/blog/`.